typeset/derive/line.satyh

```diff
@@ -1,14 +1,25 @@
 @require: gr
 
-type derivation-line = context -> point -> length -> length -> graphics list
-
 module DeriveLine : sig
 
-    val standard-line : derivation-line
-    val no-line       : derivation-line
-    val dotted-line   : length -> length -> derivation-line
+    % TODO: rename to t after the name resolution issue is resolved
+    % https://github.com/gfngfn/SATySFi/issues/185
+    type u

```

Comment:
Is it really necessary to name this u instead of t?

Reply:
Yep, I got compiler error when I put `DeriveLine.t` before I found the issue.

Reply:
I think the issue should occur when you write a type alias like
```
module Foo : sig
  type t
  ....
end = struct
  type t = Hoge (DeriveLine.t)     % define `t` using `t` from another module
  ....
end
```
I usually remedy this by making a temporary type alias `u` as follows
```
module Foo : sig
  type t
  ....
end = struct
  type u = Hoge (DeriveLine.t)
  type t = u
  ....
end
```
This way you won't have to change the name of `t`, which is exported and hard to change later. If your issue is of the same kind, could you check if this workaround works?

Reply:
It worked. Thank you for the advice 👍